### ISSUE TITLE
Unable to toggle lights on printer.

### ISSUE THREAD

**Issue by mon5termatt** [bigtreetech/K-Touch#24]
If a button for light control could be added that would be nice.

**Comment by Andy3dprinter** [comment]
I agree. I have mine running with a rooted K1 and would love the ability as well to control the LED light.

**Comment by PoohWoah** [comment]
Ok, thank you for your feedback. We will consider adding this function in the subsequent version.

**Comment by fierland** [comment]
me too
